sveltekit-directus/index.md

```diff
@@ -0,0 +1,348 @@
+
+## Before You Start
+
+You will need:
+
+- To install Node.js and a code editor on your computer.
+- To sign up for a Directus Cloud account.
+- Some knowledge of Svelte.
+
+## Create Backend (Directus)
+
+Create a new Directus Cloud project - any tier and configuration is suitable for this tutorial. You can also choose to
+install Directus [locally](/self-hosted/quickstart).
+
+## Create Frontend (SvelteKit)
+
+Open your terminal and run the following commands to create a new SvelteKit project and add the Directus JavaScript SDK:
+
+```
+npm create svelte@latest frontend (Choose Skeleton project)
+cd frontend
+npm install
+npm install @directus/sdk
+```
+
+Open `frontend` in your code editor and type `npm run dev` in your terminal to start the Vite development server and
+open http://localhost:5173 in your browser.
+
+### Create a Wrapper for the SDK
+
+We now need to setup the Directus SDK and make it accessible globally. In order to make the best use of SvelteKit's
+Server Side Rendering we will need to use SvelteKit's own
+[fetch implementation](https://kit.svelte.dev/docs/load#making-fetch-requests). Create a new file `directus.js` inside
+of the `src/libs` directory
+
+```js
+import { createDirectus, rest } from '@directus/sdk';
+import { readItems, readItem, updateItem, updateUser, createItem, deleteItem } from '@directus/sdk';
+
+function getDirectusInstance(fetch) {
+	const directus = createDirectus(import.meta.env.VITE_APIURL, { globals: { fetch } }).with(rest());
+
+  //These are optional shortcuts. Feel free to omit those and adapt the code later
+	directus.updateUser = async (id, query) => directus.request(updateUser(id, query));
+	directus.updateItem = async (collection, id, query) => directus.request(updateItem(collection, id, query));
+	directus.readItems = async (collection, query) => directus.request(readItems(collection, query));
+	directus.readItem = async (collection, id, query) => directus.request(readItem(collection, id, query));
+	directus.createItem = async (collection, query) => directus.request(createItem(collection, query));
+	directus.deleteItem = async (collection, id) => directus.request(deleteItem(collection, id));
+
+	return directus;
+}
+
+export default getDirectusInstance;
+```
+
+In order to make this work we also need to create a `hooks.server.js` file with the following content in the root
+directory. It makes sure that the required headers for fetching JavaScript content are returned by the SvelteKit Server
+
+```js
+export async function handle({ event, resolve }) {
+	return await resolve(event, {
+		filterSerializedResponseHeaders: (key, value) => {
+			return key.toLowerCase() === 'content-type';
+		},
+	});
+}
+```
+
+::: tip Directus SDK
+
+Theoretically you could also make HTTP requests to your Directus server endpoint directly via SvelteKit's `fetch`
+implementation. However the Directus SDK offers some nice [additional features](/guides/sdk/getting-started).
+
+:::
```

Comment:
however we will need to use this for authentication as the directus SDK does not play nice together with sveltekits SSR. but sure it can be removed here. i just think one should in general also explain users what the options are in order to better understand the reasoning behind choosing a path

Reply:
With that context I'm happy to leave it